app/controller/command/commands/__init__.py

```diff
@@ -1,8 +1,5 @@
 """Pack the modules contained in the commands directory."""
-from typing import Union
 from app.controller.command.commands.team import TeamCommand
 from app.controller.command.commands.user import UserCommand
 from app.controller.command.commands.token import TokenCommand
 from app.controller.command.commands.project import ProjectCommand
```

Comment:
According to CI, these aren't used anymore, and should be removed.

Reply:
I think those imports are still used by instances where those commands are imported from `app.controller.command.commands`.